runtimes/deno-1.20/server.ts

```diff
@@ -0,0 +1,55 @@
+import { Application } from "https://deno.land/x/oak@v10.1.0/mod.ts";
+import * as path from "https://deno.land/std@0.119.0/path/mod.ts";
+
+const USER_CODE_PATH = '/usr/code-start';
+
+const app = new Application();
+
+app.use(async (ctx) => {
+  const { value } = ctx.request.body({ type: 'json' });
+  const body = await value;
+
+  if (ctx.request.headers.get("x-internal-challenge") !== Deno.env.get("INTERNAL_RUNTIME_KEY")) {
+    ctx.response.status = 401;
+    ctx.response.body = {
+      code: 401,
+      message: "Unauthorized"
+    };
```

Comment:
We're no longer returning an object in case of errors. 
https://github.com/open-runtimes/open-runtimes/blob/3261573d3a02e7dbad1868e465d2858a3a183494/runtimes/deno-1.13/server.ts#L14